src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -0,0 +1,85 @@
+package com.rtb.projectmanagementtool.auth;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.users.*;
+import com.rtb.projectmanagementtool.user.*;
+import java.io.*;
+import java.util.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+
+public class AuthOps {
+
+  // private DatastoreService datastore;
+  private UserController controller;
+  public String cookieName;
+  public String cookieValue;
+  public Cookie currCookie;
+
+  public AuthOps(UserController controller) {
+    this.controller = controller;
+    cookieName = "sessionUserID";
```

Comment:
These won't change.  These can be static constants for your whole class.  Also for cookieValue, since this specifically holds the "not logged in" value, try to rename the variable as such.  Check what naming convention the team is using for static variables.